packages/styled-components/components/menu/themes/defaultVertical.js

```diff
@@ -0,0 +1,48 @@
+import styled from 'styled-components';
+
+/* eslint-disable import/prefer-default-export */
+export const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+`;
+
+// Wrapper around the menu name.
+export const NameWrapper = styled.h3`
```

Comment:
Depending on where a menu is used the required heading level can change -- keeping it out of the style sheet as a dependency is a good idea.

Reply:
This can be overridden by a developer injecting a new theme, which I believe would be the recommended solution.